### ISSUE TITLE
Be able to specify what gets focus when a tab is closed

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#5855]
## Description 
The older Muon version of Brave allowed for control over what happened when you close a tab. There was a setting `When closing an active tab:` with the values:
- `Select its parent tab` (default)
- `Select the last viewed tab`
- `Select the next tab`

<img width="322" alt="Screen Shot 2019-09-01 at 11 15 45 PM" src="https://user-images.githubusercontent.com/4733304/64093858-b156c200-cd0e-11e9-89ff-03c6779828e8.png">

<img width="301" alt="Screen Shot 2019-09-01 at 11 15 56 PM" src="https://user-images.githubusercontent.com/4733304/64093865-b4ea4900-cd0e-11e9-8292-d7e25ba6f889.png">

**Comment by bsclifton** [comment]
+1 from @dlewand691 via https://github.com/brave/brave-browser/issues/5846
> When closing a tab, just go to the next one to the right.

**Comment by dlewand691** [comment]
Yes, please, please, please! The parent tab switching is such a frustrating experience.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
